app/src/main/res/values/strings.xml

```diff
@@ -65,9 +65,13 @@
 
     <!-- Private mode shortcut "contextual feature recommendation" (CFR) -->
     <!-- Text for the main message -->
-    <string name="cfr_message">Add a shortcut to open private tabs from your Home screen.</string>
+    <string name="cfr_message" moz:removedIn="109" tools:ignore="UnusedResources">Add a shortcut to open private tabs from your Home screen.</string>
+    <!-- Text for the main message -->
```

Comment:
Could we also update the localization comments for the new strings to add a bit more detail to avoid the l10n team coming back to us to add more details. I know it's the same message, but I think we should always improve things when we can.